tests/UI/campaigns/functional/API/02_endpoints/09_product/04_patchProductId.ts

```diff
@@ -183,30 +193,210 @@ describe('API : PATCH /product/{productId}', async () => {
     });
 
     [
-      // @todo : https://github.com/PrestaShop/PrestaShop/issues/35626
+      {
+        propertyName: 'active',
+        propertyValue: false,
+      },
+      {
+        propertyName: 'additionalShippingCost',
+        propertyValue: faker.number.float({fractionDigits: 2}),
+      },
+      // @todo : https://github.com/PrestaShop/PrestaShop/issues/38787
       /*{
-        propertyName: 'type',
-        propertyValue: patchProduct.type,
+        propertyName: 'availableDate',
+        propertyValue: faker.date.future(),
       },*/
       {
-        propertyName: 'active',
-        propertyValue: patchProduct.status,
+        propertyName: 'availableForOrder',
+        propertyValue: false,
       },
       {
-        propertyName: 'names',
+        propertyName: 'availableLaterLabels',
+        propertyValue: {
+          [dataLanguages.english.locale]: faker.lorem.sentence(),
+          [dataLanguages.french.locale]: faker.lorem.sentence(),
+        },
+      },
+      {
+        propertyName: 'availableNowLabels',
         propertyValue: {
-          [dataLanguages.english.locale]: patchProduct.name,
-          [dataLanguages.french.locale]: patchProduct.nameFR,
+          [dataLanguages.english.locale]: faker.lorem.sentence(),
+          [dataLanguages.french.locale]: faker.lorem.sentence(),
         },
       },
+      {
+        propertyName: 'condition',
+        propertyValue: 'used',
+      },
+      {
+        propertyName: 'deliveryTimeInStockNotes',
+        propertyValue: {
+          [dataLanguages.english.locale]: faker.lorem.sentence(),
+          [dataLanguages.french.locale]: faker.lorem.sentence(),
+        },
+      },
+      {
+        propertyName: 'deliveryTimeOutOfStockNotes',
+        propertyValue: {
+          [dataLanguages.english.locale]: faker.lorem.sentence(),
+          [dataLanguages.french.locale]: faker.lorem.sentence(),
+        },
+      },
+      {
+        propertyName: 'deliveryTimeNoteType',
+        propertyValue: 2,
+      },
+      {
+        propertyName: 'depth',
+        propertyValue: faker.number.float({fractionDigits: 2}),
+      },
       {
         propertyName: 'descriptions',
         propertyValue: {
-          [dataLanguages.english.locale]: patchProduct.description,
-          [dataLanguages.french.locale]: patchProduct.descriptionFR,
+          [dataLanguages.english.locale]: faker.lorem.sentence(),
+          [dataLanguages.french.locale]: faker.lorem.sentence(),
         },
       },
-    ].forEach((data: { propertyName: string, propertyValue: boolean|string|object}) => {
+      // @todo : https://github.com/PrestaShop/PrestaShop/issues/38788
```

Comment:
That's because obsolete `ean13` was replaced with `gtin` (equivalent to `ean14` I believe). The ean is deprecated and will be removed in 10.0, but since it's still here and the serialization is automatized both fields are present

You can use one or the other they both should work, but when you use both at the same time since the value ends up in the same class field t naturally overwrites the first one

Reply:
But it's not deprecated in the API : you can define it in API Platform (https://api-platform.com/docs/core/deprecations/#deprecating-resource-classes-operations-and-properties)

I have created an issue with info : https://github.com/PrestaShop/PrestaShop/issues/38788.